service_crategen/src/main.rs

```diff
@@ -261,6 +261,18 @@ See [LICENSE][license] for details.
 //!
 //! If you're using the service, you're probably looking for [{client_name}](struct.{client_name}.html) and [{trait_name}](trait.{trait_name}.html).
 
+// =================================================================
```

Comment:
Agreed.  @kpcyrd can you move this to the top of this string so it's the first thing in the generated files?  Thanks!